sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClientIntegrationTest.java

```diff
@@ -659,90 +716,90 @@ void receiveAndRenewLock(MessagingEntityType entityType) {
     @ParameterizedTest
     void autoRenewLockOnReceiveMessage(MessagingEntityType entityType, boolean isSessionEnabled) {
         // Arrange
-        setSenderAndReceiver(entityType, 0, isSessionEnabled);
+        final AtomicInteger lockRenewCount = new AtomicInteger();
+
+        setSender(entityType, TestUtils.USE_CASE_DEFAULT, isSessionEnabled);
 
         final String messageId = UUID.randomUUID().toString();
         final ServiceBusMessage message = getMessage(messageId, isSessionEnabled);
 
-        if (isSessionEnabled) {
-            message.setSessionId(sessionId);
-        }
```

Comment:
` getMessage(messageId, isSessionEnabled);` Already set session id , we do not need this again.